binary_search_trees/array_to_bst.py

```diff
@@ -10,4 +10,62 @@ def arr_to_bst(arr):
         Balanced Binary Search Tree using the elements in the array.
         Return the root of the Binary Search Tree.
     """
-    pass
\ No newline at end of file
+
+    class BinarySearch_tree:
```

Comment:
It's a bit funky to define a class within a function. I'm not sure that it's _not allowed_ but I believe it does create some weird situations with Python's rules about variable/object scope. We can actually create a binary search tree even without creating a `BinarySearchTree` class. 

Notice from the Learn lessons (and your implementation!) that the `BinarySearchTree` class simply bundles together a single piece of data `root` with some functions. The `root` actually _is_ the tree, in that it's the topmost node in the tree and via the child pointers we can reach all the other nodes in the tree. 

We can uncouple the `root` and class methods, and simply use a variable `root` to represent the tree. Instead of class methods where the tree is passed in via `self`, each of our functions would instead need to have an explicit parameter `root` (or similar) to pass in the tree.